packages/subgraph/src/resolver.ts

```diff
@@ -145,6 +155,47 @@ export function handleContentHashChanged(event: ContenthashChangedEvent): void {
   resolverEvent.save()
 }
 
+export function handleDNSRecordChanged(event: DNSRecordChangedEvent): void {
```

Comment:
Yup, ens-sugraph doesn't have DNS events. If this works, then can maybe PR to their repo :p

Reply:
Have you tried creating an API request to get DNS records for a domain? I'm playing around with the subgraph on mumbai and it seems there is no way to filter Resolver events by type and records aren't stored at top level Domain either. So we'd have to pull in all resolver events for a domain and then filter on front-end for DNS records? 

Might be worth considering creating a `Record` entity in schema and creating field `Domain.records: [Record!]`. Based on ENS setup I think Record would have to be an RRSet not an individual DNS record. Can still keep all the ResolverEvents just have them update the Record entity and save to Domain for easier access.

I could be totally wrong though, let me know what you think

Reply:
DNS records are submitted to the resolver contract as multiple DNS wire format strings. The contract decodes each name and stores it by hash so there's no way to get all records (for different names) unless you specify exactly what "name" and "type" to get records for ("sh8.", "A").

So yeah, subgraphs aren't useful to extract dns records so these events are nice to have to keep track of what was added (since the full list cannot be retrieved from the resolver at all).

Same goes for Text records. Subgraphs only store keys, not values. ENS app gets these keys from the subgraph and queries the resolver to get values for each key.

Reply:
About storing parsed DNS records on subgraph, would be great if possible but it's assemblyscript and not sure if bns (required to parse dns wire format) can be used in event handlers.

Reply:
Thinking now, it may be possible to modify DNSResolver to emit keys (dns record names as strings and use it in the same way as how TXT records are used (prev comment)

Reply:
I think it should be possible since everything gets built and bundled before being sent to the graph indexers to run. Like we have `@ensdomains/content-hash` package included in our subgraph code which isn't from official Graph libraries and all packages in subgraph are listed as dev dependencies. 

If not possible its not a problem, huge improvement already.